### ISSUE TITLE
is `--disable-multithreading` a valid parameter for prefetch?

### ISSUE THREAD

**Issue by wisekh6** [ncbi/sra-tools#346]
Hi,

According the prefetch's help, `--disable-multithreading` looks a valid parameter, but it is not recognized when I try to run.
Would you help me with how to disable multithreading for prefetch?

Thank you in advance,

```
prefetch --disable-multithreading --max-size 100000000 SRR8639240 && fasterq-dump --threads 4 SRR8639240

2020-06-03T19:06:34 prefetch.2.10.7 err: param unknown while parsing argument list within application support module - Unknown argument '--disable-multithreading'
2020-06-03T19:06:34 prefetch.2.10.7 int: param unknown while parsing argument list within application support module - ArgsMakeAndHandle
```

```
$ prefetch --help

Usage: prefetch [ options ] [ accessions(s)... ]

Parameters:

  accessions(s)                    list of accessions to process


Options:

  -T|--type <file-type>            Specify file type to download. Default: sra
  -N|--min-size <size>             Minimum file size to download in KB
                                     (inclusive).
  -X|--max-size <size>             Maximum file size to download in KB
                                     (exclusive). Default: 20G
  -f|--force <no|yes|all|ALL>      Force object download - one of: no, yes,
                                     all, ALL. no [default]: skip download if
                                     the object if found and complete; yes:
                                     download it even if it is found and is
                                     complete; all: ignore lock files (stale
                                     locks or it is being downloaded by
                                     another process - use at your own
                                     risk!); ALL: ignore lock files, restart
                                     download from beginning
  -p|--progress                    Show progress
  -r|--resume <yes|no>             Resume partial downloads - one of: no, yes
                                     [default]
  -C|--verify <yes|no>             Verify after download - one of: no, yes
                                     [default]
  -c|--check-all                   Double-check all refseqs
  -o|--output-file <file>          Write file to <file> when downloading
                                     single file
  -O|--output-directory <directory>
                                   Save files to <directory>/
     --ngc <path>                  <path> to ngc file
     --perm <path>                 <path> to permission file
     --location <location>         location in cloud
     --cart <path>                 <path> to cart file
     --disable-multithreading      disable multithreading
  -V|--version                     Display the version of the program
  -L|--log-level <level>           Logging level as number or enum string.
                                     One of
                                     (fatal|sys|int|err|warn|info|debug) or
                                     (0-6) Current/default is warn
     --option-file file            Read more options and parameters from the
                                     file.
  -h|--help                        print this message

"prefetch" version 2.10.7
```

**Comment by benostendorf** [comment]
The same seems to be true for `fasterq-dump`. Thanks for looking into it!

**Comment by klymenko** [comment]
It's a bug. Thank you for reporting!

**Comment by klymenko** [comment]
It was fixed in 2.11.0 release. Please confirm.
